<launch>
  <node name="cross_detector" pkg="cross_detector" type="cross_detector_node" output="screen" >
     <remap from="camera" to="/uav4/bluefox/image_raw" />
     <remap from="exposure_time" to="/mv_bf_down/exposure_time" />
     <param name="target_frame" type="string" value="local_origin"/>
     <param name="pattern_expected_altitude" type="double" value="0.0"/>
     <param name="gui" type="bool" value="true"/>
     <param name="calib" type="bool" value="false"/>
     <param name="camera_info" type="bool" value="false"/>
     <param name="image_transport" type="string" value="compressed" />
     <param name="frame_topic" type="string" value="/fcu_uav4" />
</node>
</launch>


    

  